test-app/src/androidTest/java/com/linkedin/parser/test/junit4/java/BasicJUnit4.java

```diff
@@ -8,9 +8,11 @@
 
 import static org.junit.Assert.assertTrue;
 
+@TestValueAnnotation("Hello world!")
 public class BasicJUnit4 {
 
     @Test
+    @TestValueAnnotation("On a method")
```

Comment:
it would be nice to see other tests with:
 - Something else rather than string
 - More than one key, eg: `@TestAnnotation(key1="value1", key2=1234)`

Reply:
Good idea, I'll update!